db/madmin_conversion.py

```diff
@@ -0,0 +1,306 @@
+TABLES = [
+    """CREATE TABLE IF NOT EXISTS `madmin_instance` (
+        `instance_id` INT NOT NULL AUTO_INCREMENT,

```

Comment:
Consider having all primary and foreign keys `int unsigned`.

Reply:
Agreed.  Resolved in 46ad21a